src/App.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import UserInput from "./components/UIComponents/UserInput";
+import UsersList from "./components/UIComponents/UsersList";
+import { useState } from "react";
+import ErrorModal from "./components/UIComponents/ErrorModal";
+
+function App() {
+  const initialCondition = { flag: true, msg: "" };
+  const [isValid, setIsvalid] = useState(initialCondition);
+  const [userInput, setUserInput] = useState([]);
+  const userDataHanler = (userInput) => {
+    console.log(userInput);
+    setUserInput((prev) => {
+      if (userInput["user-name"] === "" || userInput["user-age"] === "") {
+        setIsvalid({
+          flag: false,
+          msg: "Please, fill all the data (no value should be empty)",
+        });
+        console.log(isValid);
+        return prev;
+      }
+      if (userInput["user-age"] <= 0) {
```

Comment:
age should not be more than 100 so you can use this condition to restrict user to put age more than 100

Reply:
sure brother!
let me do it

Reply:
its done

Reply:
but don't forget to use delet functionality as I suggested 😁